### ISSUE TITLE
allenai/longformer-base-4096 not suited for sentiment classification

### ISSUE THREAD

**Issue by llinauer** [taranis-ai/sentiment-analysis#2]
Maybe you already know this, but the `allenai/longformer-base-4096` model is not trained for sentiment classification.
If you used it with the transformers.LongformerForSequenceClassification, the weights of the classification head are randomly initialized and the classified sentiment will be more or less random.

A quick research did not yield any pre-trained sentiment analysis Longformer. 

### Details

This warning is shown when creating an instance of the SentimentAnalysis class

```
>>> from sentiment_analysis.sentiment_analysis_multimodel import SentimentAnalysis
>>> se = SentimentAnalysis()
Some weights of LongformerForSequenceClassification were not initialized from the model checkpoint at
allenai/longformer-base-4096 and are newly initialized:
['classifier.dense.bias', 'classifier.dense.weight', 'classifier.out_proj.bias', 'classifier.out_proj.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
```

**Comment by llinauer** [comment]
I'm gonna remove the longformer from the repo and instead use a majority vote technique for the roberta model to process longer texts.
The idea is to split a long text into chunks, feed all of them in a batch through Roberta and then take the majority of predicted sentiments as the sentiment of the full text.

I ran tests on two test sets:
- One containing only short texts with < 128 tokens
- One containing longer and shorter texts combined (< 1325 tokens)

Both datasets contain 400 items, 200 english and 200 german

## Tests

### Roberta with majority vote vs. unbatched

I checked if the majority vote method drastically worsens performance.
I tested on the *short* dataset and evaluated for balanced accuracy and macro-averaged F1 score. The results are virtually identical:
Roberta unbatched: Accuracy: 0.630,  F1: 0.629
Roberta with Majority vote: Accuracy: 0.626,  F1: 0.629

Due to batching, the runtime of both is also identical:

Roberta with majority vote : 0.06s / text
Roberta unbatched: 0.07s / text


### Roberta with majority vote vs. Longformer

I tested the majority vote method on the *long* dataset and compared it with the Longformer:
Roberta with majority vote: Accuracy: 0.659,  F1: 0.723
Longformer: Accuracy: 0.307,  F1: 0.382

So, as expected, the Longformer is way worse. It basically only ever predicts "neutral"

I will implement the changes, once https://github.com/taranis-ai/sentiment-analysis/pull/6 is merged
